cleanupJavadoc.sh

```diff
@@ -1,2 +0,0 @@
-find release/doc -name "*.html" -exec sed -i "s/.*Generated by javadoc.*//" {} \;
```

Comment:
Sorry, probably I missed some point where the docs were generated or something like that.